e2e/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+# color codes
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+function getPassboltSecret() {
+    local apiVersion=$1
+    kubectl get \
+        passboltsecrets.${apiVersion}.passbolt.tagesspiegel.de \
+        passboltsecret-sample-${apiVersion} \
```

Comment:
```suggestion
        "passboltsecret-sample-${apiVersion}" \
```

Reply:
We are executing a command against the K8s api. Bash completion would never suggest that you reference it with quotes